files/image_config/caclmgrd/caclmgrd

```diff
@@ -16,7 +16,10 @@ try:
     import subprocess
     import sys
     import syslog
-    from swsssdk import ConfigDBConnector
+
+    import sonic_device_util
```

Comment:
Suggest to move to the new sonic-py-common package.

```
from sonic_py_common import device_info
```

Reply:
@jleveque Done